nifi-mock/src/test/java/org/apache/nifi/util/TestMockProcessSession.java

```diff
@@ -16,6 +16,9 @@
  */
 package org.apache.nifi.util;
 
+import java.util.Collections;
+import java.util.Set;
+
```

Comment:
I believe your IDE is automatically changing the import order when you save your modifications. Could you revert the changes?